### ISSUE TITLE
Issue/Bug with hasperm-statement in Version 3.7.0/3.7.1/3.7.2 | Permissions aren't checked by CP

### ISSUE THREAD

**Issue by Endphoenix** [rockyhawk64/CommandPanels#39]
### Have you checked the [wiki](https://rockyhawk99.gitbook.io/rockyhawk-wiki/commandpanels/wiki) page to see if the solution for your problem is there?
Yes

### Have you checked the [issue tracker](https://github.com/rockyhawk64/CommandPanels/issues?q=is%3Aissue) to see if your issue is a duplicate and your problem already has been solved?
Yes

### Have you asked for help in the [Discord Server](https://discord.com/invite/eUWBWh7)? *(Or got redirected to a Github issue here)*
Yes

### Describe the Issue
A clear and concise description of what the problem is.

```
 '10':
        material: BRICK
        name: '&7Level &e1'
        lore:
        - '&7You can have &e3 Enchantments &7per Item.'
        - '&7Price: &a1000 coins'
        commands:
        - paywall= 1000
        - console= lp user %player_name% permission set ae.limit.3 true
        - console= lp user %player_name% permission set commandpanel.enchantment.level1 true
        hasperm:
          output: true
          perm: commandpanel.enchantment.level1
          material: BRICK
          name: '&7Level &e1'
          lore:
          - '&7You can have &e3 Enchantments &7per Item.'
          - '&aUnlocked'
          enchanted: true

```
If I open as the panel without the Permssion "commandpanel.enchantmant.level1" I should actually see a Brick with the description "&7You can have &e3 Enchantments &7per Item &7Price: &a1000 coins". That is also the case. But as soon as I click on the Brick with enough money I should get the permission "commandpanel.enchantmant.level1". So I should see an enchanted Brick with another Lore. Instead, the GUI doesn't change and I can buy the level over and over again. I don't understand what I'm doing wrong. To clarify the video below:

https://imgur.com/a/XhZlpC5

### Expected behavior

Can you say me what I am doing wrong?

### Screenshots/videos
If applicable, add screenshots or videos to help explain your problem.

https://imgur.com/a/XhZlpC5

### Versions:
 - Minecraft: Spigot 1.14.4
 - CommandPanels: 3.7.2

### Additional context/Panel files
Add any other context about the problem here. Please also send you're panel in question. We only accept [pastebin](https://pastebin.com/) links.

https://pastebin.com/JQ0gTwzd

/\
 | 

I've only done Level 1 (Brick) to test if that's possible at all.

**Comment by TheLonelyWolf1** [comment]
Hello.
I'm now testing your problem and see if i can identify it ;)

**Comment by TheLonelyWolf1** [comment]
After some testing i found out that there is a bug with hasperm in the affecting multiple CP-Versions.

Currently this Versions are identified to have your issue:
- Command Panels 3.7.2
- Command Panels 3.7.1
- Command Panels 3.7.0

The Issue doesn't seems to appear in Version 3.6.0. Please downgrade to that Version while the dev will try to identify the problem inside the code.

**Comment by TheLonelyWolf1** [comment]
The Problem seems to be resolved in Version 3.7.4. Please update to that version and see if it works for you now.
